arcgis-ios-sdk-samples/Maps/Display layer view state/LayerStatus.storyboard

```diff
@@ -97,8 +107,7 @@
                     </view>
                     <connections>
                         <outlet property="mapView" destination="MMu-eo-dCE" id="mFj-mZ-uMd"/>
-                        <outlet property="tableView" destination="YVw-Ng-UNh" id="eto-31-LmG"/>
-                        <outlet property="visualEffectView" destination="zsC-4Y-Zj7" id="b8q-ZC-v9e"/>
+                        <outlet property="statusLabel" destination="ZtX-rV-bcZ" id="OfY-yE-fgj"/>
```

Comment:
What do you think about putting the status label next to the switch on the toolbar? I think it could utilize some unused space since there's only one item in the toolbar.

Reply:
For the smallest device still in support (iPhone SE 1st gen) it might be a little bit packed up if to put it there. But yeah this is a good suggestion and let's see what Phil says.

I tend to put the status message bar atop to be consistent with earlier samples designs, so that users can intuitively find the status messages.

![image](https://user-images.githubusercontent.com/9660181/89589829-5c171080-d7fb-11ea-811d-0a6b9d1deabb.png)